rectools/models/nn/transformers/torch_backbone.py

```diff
@@ -168,11 +179,34 @@ def forward(
         ----------
         sessions : torch.Tensor
             User sessions in the form of sequences of items ids.
+        TODO
 
         Returns
         -------
-        (torch.Tensor, torch.Tensor)
+        (torch.Tensor, torch.Tensor, torch.Tensor)
         """
         item_embs = self.item_model.get_all_embeddings()  # [n_items + n_item_extra_tokens, n_factors]
         session_embs = self.encode_sessions(sessions, item_embs)  # [batch_size, session_max_len, n_factors]
-        return item_embs, session_embs
+        logits = self.similarity_module(session_embs, item_embs, item_ids)
+        if last_n_items:
+            logits = logits[:, -last_n_items:, :]
+        return item_embs, session_embs, logits
+
+    def _recommend_u2i(
```

Comment:
Do we really need it? We can just call `self.torch_model.similarity_module._recommend_u2i` instead

Reply:
Fixed